docs/en/observability/synthetics-reference/lightweight-config/http.asciidoc

```diff
@@ -0,0 +1,305 @@
+a| The HTTP proxy URL. This setting is optional.
+
+*Example*:
+
+[source,yaml]
+http://proxy.mydomain.com:3128
+
+////////////////////////
+username
+////////////////////////
+| [[monitor-http-username]] *`username`*
+(<<synthetics-lightweight-data-string,string>>)
+a| The username for authenticating with the server. The credentials are passed with the request. This setting is optional.
+
+You need to specify credentials when your `check.response` settings require it. For example, you can check for a 403 response (`check.response.status: [403]`) without setting credentials.
+
+////////////////////////
+password
+////////////////////////
+| [[monitor-http-password]] *`password`*
+(<<synthetics-lightweight-data-string,string>>)
+a| The password for authenticating with the server. This setting is optional.
+
+////////////////////////
+ssl
+////////////////////////
+| [[monitor-http-ssl]] *`ssl`*
+({heartbeat-ref}/configuration-ssl.html[SSL])
+a| The TLS/SSL connection settings for use with the HTTPS endpoint. If you don't specify settings, the system defaults are used.
+
+*Example*:
+
+[source,yaml]
+----
+- type: http
+  id: my-http-service
+  name: My HTTP Service
+  hosts: ["https://myhost:443"]
+  schedule: '@every 5s'
+  ssl:
+    certificate_authorities: ['/etc/ca.crt']
+    supported_protocols: ["TLSv1.0", "TLSv1.1", "TLSv1.2"]
+----
+
+////////////////////////
+headers
+////////////////////////
+| [[monitor-http-headers]] *`headers`*
+(<<synthetics-lightweight-data-bool,boolean>>)
+a| Controls the indexing of the HTTP response headers `http.response.body.headers` field. Set `response.include_headers` to `false` to disable.
+
+*Default*: `true`
+
+////////////////////////
+response
+////////////////////////
+| [[monitor-http-response]] *`response`*
+a| Controls the indexing of the HTTP response body contents to the `http.response.body.contents` field.
+
+////////////////////////
+response.include_body
+////////////////////////
+*`response.include_body`* (`"on_error"` \| `"never"` \| `"always"`)::
+Set `response.include_body` to one of the options listed below.
++
+* `on_error`: Include the body if an error is encountered during the check. This is the default.
+* `never`: Never include the body.
+* `always`: Always include the body with checks.
+
+////////////////////////
+check
+////////////////////////
+| [[monitor-http-check]] *`check`*
+a| 
+////////////////////////
+// check.request
+////////////////////////
+*`request`*:: An optional `request` to send to the remote host. Under `check.request`, specify these options:
++
+--
+////////////////////////
+// check.request.method
+////////////////////////
+*`method`* (`"HEAD"` \| `"GET"` \| `"POST"` \| `"OPTIONS"`)::: The HTTP method to use.
+
+////////////////////////
+// check.request.headers
+////////////////////////
+*`headers`* (?):::
+  A dictionary of additional HTTP headers to send. By default heartbeat will set the 'User-Agent' header to identify itself.
```

Comment:
Something like:

```
headers:
  X-Favorite-Robot: bender
```

would be a fine inline example. I don't think a link out to MDN would hurt, but I'm not sure it's necessary either since I assume folks looking here know about HTTP headers (though I could be wrong!)